src/utils/hints/tokensHints.js

```diff
@@ -0,0 +1,5 @@
+export const tokenHints = {
+  tokenSymbol: 'Symbol of a token. Symbol does not have to be unique.',
+  tokenName: 'Full name of a token.',
```

Comment:
the token?

Reply:
here is disputable, but I went with `the`.